sanitize.go

```diff
@@ -259,6 +262,13 @@ func (p *Policy) sanitizeAttrs(
 	// whitelisted explicitly or globally.
 	cleanAttrs := []html.Attribute{}
 	for _, htmlAttr := range attrs {
+		if p.allowDataAttributes {
+			// If we see a data attribute, let it through.
+			if dataAttribute.MatchString(htmlAttr.Key) {
+				cleanAttrs = append(cleanAttrs, htmlAttr)
```

Comment:
I read the HTML5 spec, and it's crazy... this attribute can literally contain "any value".

2GB base64 encoded video file? Sure... it's valid. 🤦‍♂️  But that is what the spec says so this code is technically fine 👍